FIAT/barycentric_interpolation.py

```diff
@@ -0,0 +1,35 @@
+# This is work in progress
+
+import numpy
+
+
+def barycentric_interpolation(xsrc, xdst, order=0):
+    """Returns a dict with differentiation matrices up to a given order
```

Comment:
Could you please add a docstring that describes the arguments with appropriate rst formatting. Prefer the active "Return a dict" rather than "returns a dict"

I think this returns tabulations?

```
Return tabulations of ...

:arg xsrc: ...
:arg xdst: ...
:arg order: ..
:returns: dict of tabulations up to the given order (in the same format as :meth:`~.CiarletElement.tabulate`)
```

Or something like that.